README.md

```diff
@@ -100,6 +100,7 @@ Finally, enable all of the rules that you would like to use.  Use [our preset](#
 * [react/no-redundant-should-component-update](docs/rules/no-redundant-should-component-update.md): Prevent usage of `shouldComponentUpdate` when extending React.PureComponent
 * [react/no-render-return-value](docs/rules/no-render-return-value.md): Prevent usage of the return value of `React.render`
 * [react/no-set-state](docs/rules/no-set-state.md): Prevent usage of `setState`
+* [react/no-static-typos](docs/rules/no-static-typos.md): Prevent casing typos when declaring static class properties.
```

Comment:
Agreed. I will rename to `no-typos` and update the docs accordingly. 

We can then add casing typos for lifecycle methods in another PR. And we can look into detecting small Levenshtein distances in a separate PR as well. 

Sounds good?